docs/tutorials/sui-decompiler/index.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Dark Side of Sui Move Decompiler
```

Comment:
How about **"Challenges and Limitations of Sui Move Decompiler in Smart Contract Verification"**?